### PR TITLE
add support for txt2img task

### PR DESCRIPTION
**Pull request by jeffra**
Adds support for txt2img task in MII. This feature is dependent on the DS side PR: https://github.com/microsoft/DeepSpeed/pull/2381

### CODE REVIEW

File: mii/models/load_models.py [hunk 1]
```diff
@@ -27,6 +27,8 @@ def load_models(task_name,
     mpu = None
     args = None
     training_mp_size = 1
+    kernel_inject = True
+    replace_pipeline_model = True
```
Comment (mrwyattii) [inline mii/models/load_models.py:31]: This seems to be an unused variable

**Review by mrwyattii — APPROVED** [review]
LGTM - should we also refactor the other examples to include the example/query in a single script?

File: mii/models/load_models.py (hunk 1 above)
Comment (jeffra) [reply mii/models/load_models.py]: good catch, fixed :)

**Review by awan-10 — APPROVED** [review]
Looks good!

**Pull request merged**